### DISCUSSION TITLE
Nullable Types

### DISCUSSION THREAD

**Discussion by mishelshaji** [mishelshaji/Learn-DotNet#8] [Progress Tracker]
Topics covered:
👉 What is a null value?
👉 null vs empty string.
👉 Nullable and non-nullable types.
👉 Checking if a variable is null.
👉 Null reference exception.
👉 Creating basic types like int as nullable.
👉 Nullable type changes in old and new versions of C#.
👉 Turning off the null warning.

**Comment by ManikuttyBabu** [comment]
completed

**Comment by Arjunprasad-22** [comment]
Completed

**Comment by PEFathimaManfa** [comment]
completed

**Comment by Arun-babumon** [comment]
Completed

**Comment by sreelakshmiChandran** [comment]
Completed

**Comment by Nandana-04** [comment]
Completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by Jiby-Joseph-123** [comment]
Completed.

**Comment by riya45810** [comment]
Completed.

**Comment by SivapriyaVB** [comment]
Completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by jeweljose22** [comment]
Completed

**Comment by AshinDamodaran** [comment]
completed

**Comment by faznan3nazer** [comment]
*Completed*

**Comment by kavitha-BS** [comment]
Compleated

**Comment by AnitaMaryThomas** [comment]
Completed

**Comment by ARAVIND0407** [comment]
completed
